src/openrct2/util/util.c

```diff
@@ -400,24 +400,32 @@ bool str_is_null_or_empty(const char *str)
     return str == NULL || str[0] == 0;
 }
 
-uint32 srand0, srand1, srand2, srand3;
+//uint32 srand0, srand1, srand2, srand3;
 
 void util_srand(sint32 source) {
+    srand(source);
+    
+    /* Old implementation, use for specific platforms when srand() causes problems.
```

Comment:
Yes you're probably right, I just thought if it's causing problems again soon it could be added back easily. I'll remove it.